Code.gs

```diff
@@ -41,6 +41,7 @@ var addCalToTitle = false;             // Whether to add the source calendar to
 var addAttendees = false;              // Whether to add the attendee list. If true, duplicate events will be automatically added to the attendees' calendar.
 var defaultAllDayReminder = -1;        // Default reminder for all day events in minutes before the day of the event (-1 = no reminder, the value has to be between 0 and 40320)
                                        // See https://github.com/derekantrican/GAS-ICS-Sync/issues/75 for why this is neccessary.
+var overrideVisibility = false;           // Changes the visibility of the event ("default", "public", "private"). Anything else will revert to the class value of the ICAL event.
```

Comment:
Thanks for updating this. Even though it doesn't matter much from the perspective of someone familiar with Javascript, let's change the default value from `false` to a string (like `""` you had before) to make it more obvious to users. I feel like if it should be set to a string value, the default value should probably also be a string

Reply:
OK